### PR TITLE
Fix: wallets currency item consistency when the currency icon_url exists

### PR DESCRIPTION
**Pull request by seanyellowd**
https://jira.openware.work/browse/EMI-423

### CODE REVIEW

File: src/components/WalletItem/index.tsx [hunk 1]
```diff
@@ -44,7 +44,11 @@ export const WalletItem: React.FunctionComponent<Wallet> = (props: Wallet) => {
     return (
         <div style={style} className={cName}>
             <div className="cr-wallet-item__info">
-                {iconUrl ? (<img alt="" className="cr-wallet-item__image-icon" src={iconUrl} />) : (<CryptoIcon className="cr-wallet-item__icon" code={currency.toUpperCase()} />)}
+                {iconUrl ? (
+                    <span className="cr-crypto-icon cr-wallet-item__icon">
+                        <img alt="" src={iconUrl} />
```
Comment (josadcha) [inline src/components/WalletItem/index.tsx:49]: put alt prop as currency name

**Review by josadcha — APPROVED** [review]

File: src/components/WalletItem/index.tsx [hunk 2]
```diff
@@ -44,7 +44,11 @@ export const WalletItem: React.FunctionComponent<Wallet> = (props: Wallet) => {
     return (
         <div style={style} className={cName}>
             <div className="cr-wallet-item__info">
-                {iconUrl ? (<img alt="" className="cr-wallet-item__image-icon" src={iconUrl} />) : (<CryptoIcon className="cr-wallet-item__icon" code={currency.toUpperCase()} />)}
+                {iconUrl ? (
+                    <span className="cr-crypto-icon cr-wallet-item__icon">
+                        <img alt={currency.toUpperCase()} src={iconUrl} />
+                    </span>
```
Comment (akhlopiachyi) [inline src/components/WalletItem/index.tsx:50]: Why did you add this span?
You can apply className attribute directly to img tag

Comment (seanyellowd) [reply src/components/WalletItem/index.tsx:50]: span exists in CryptoIcon component.
I wanted to modify less as possible for existing consistency.
I CAN remove span from both, SHOULD I?

Comment (akhlopiachyi) [reply src/components/WalletItem/index.tsx:50]: I see
No, don't do that

**Review by akhlopiachyi — APPROVED** [review]
